wgsl/index.bs

```diff
@@ -1279,15 +1279,17 @@ An attribute [=shader-creation error|must not=] be specified more than once per
 <table class='data'>
   <caption>Attributes defined in WGSL</caption>
   <thead>
-    <tr><th>Attribute<th>Valid Values<th>Description
+    <tr><th>Attribute<th>Parameters<th>Description
   </thead>
 
   <tr><td><dfn noexport dfn-for="attribute">`align`</dfn>
-    <td>[=shader-creation error|Must=] be a [=const-expression=] that [=type rules|resolves=] to an [=i32=] or [=u32=].<br>
-        [=shader-creation error|Must=] be positive.
-    <td>[=shader-creation error|Must=] only be applied to a member of a [=structure=] type.
+    <td>(*alignment*)
```

Comment:
I find the overloads a bit confusing. It didn't become obvious of the purpose until I got to `interpolate` (the first multiple parameter attribute).

Reply:
yeah. It's not clear to me that naming these params helps much .

I put them in because I think we lost something visually when removing the explicit spelling out of what the grammar rule looks like.